components/cluster/command/root.go

```diff
@@ -49,12 +49,21 @@ var (
 	skipConfirm bool
 )
 
+func scrubClusterName(n string) string {
+	return "cluster_" + telemetry.HashReport(n)
+}
+
 func getParentNames(cmd *cobra.Command) []string {
 	if cmd == nil {
 		return nil
 	}
 
 	p := cmd.Parent()
+	// always use 'cluster' as the root command name
+	if cmd.Parent() == nil {
+		return []string{"cluster"}
```

Comment:
It's better to use `cliutil.OsArgs0()` if the `cmd.Parent() == nil`.

Reply:
https://github.com/pingcap/tiup/blob/cb5bcd4fadb1521fbad4df5d1032b0e35fd355f9/pkg/cliutil/cliutil.go#L45
then we will always get `tiup`  if run by tiup.  

I think what you want is os.Args[0], the program name of cluster. keep literal "cluster" here should be ok here.

Reply:
correct： `cliutil.OsArgs0()` is "tiup cluster" but not just "tiup", but the name of root cmd is "tiup"